web/src/components/core/Installation.jsx

```diff
@@ -21,10 +21,9 @@
 
 import React from "react";
 import { InstallationProgress, InstallationFinished } from "~/components/core";
-import { IDLE } from "~/client/status";
 
-function Installation({ status }) {
-  return status === IDLE ? <InstallationFinished /> : <InstallationProgress />;
+function Installation({ isBusy }) {
+  return isBusy ? <InstallationFinished /> : <InstallationProgress />;
```

Comment:
NP: out of the scope of the PR, but you might try to move small components and their tests to TypeScript.

Reply:
I would prefer to do that on a separate PR.